src/main/java/org/myrobotlab/service/InMoovEyelids.java

```diff
@@ -210,14 +200,6 @@ public void waitTargetPos() {
   // FIXME - releasePeers()
   public void release() {
     disable();

```

Comment:
we should remove this method.. and just use enable/disable.  if the release() method is intended to be releaseService.. then references should be updated to call releaseService instead.